pkg/version.mk

```diff
@@ -15,6 +15,10 @@ define VERSION_LDFLAGS
 $(shell $(CMD_LINKFLAGS) -pkg=$(GOPATH)/src/$(1))
 endef
 
+define VERSION_CUSTOM_FLAGS
+$(shell $(CMD_LINKFLAGS) -pkg=$(firstword $(subst :, ,$(1))))
```

Comment:
I added this trick to trim an empty string from the arg. I was calling this function 

```
INSTALL_FLAGS=-ldflags='$(STRIP_DEBUG) $(call VERSION_CUSTOM_FLAGS, /devices)'
```

With a space after colon and the function uses ' ' as a first argument. So just in case anyone else does it this will save a couple of hours ))